src/general/OptionsMenu.cs

```diff
@@ -732,13 +745,28 @@ private void UpdateResetSaveButtonState()
         saveButton.Disabled = result;
     }
 
-    private void LoadLanguages(OptionButton optionButton)
+    private void UpdateDefaultAudioOutputDeviceText(OptionButton audioOutputDeviceOptionButton)
```

Comment:
Why not just directly access the option button instead of passing it?

Reply:
Seems like LoadLanguages happens to do the same thing...

Reply:
@FuJa0815 Everything else looks good, but it would probably be less confusing if we could change the option buttons to be accessed directly from variables instead of passing them in the methods.